scripts/cluster.lic

```diff
@@ -0,0 +1,179 @@
+require "dcell"
```

Comment:
I don't think you meant to include this file.

Reply:
Ops, early morning hacking problems. Thanks for spotting this